### PR TITLE
dnsdistdist: make incfiles and populate work on sh

### PR DESCRIPTION
**Pull request by pieterlexis**
And don't error when the m4 dir already exists

### CODE REVIEW

File: pdns/dnsdistdist/populate [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 set -e
 set -x
```
Comment (ekohl) [inline pdns/dnsdistdist/populate]: isn't set a bashism?

Comment (Habbie) [inline pdns/dnsdistdist/populate]: `set -e` and `set -x` are in POSIX http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html

Comment (ekohl) [inline pdns/dnsdistdist/populate]: TIL

**Comment by rubenk** [comment]
or use sh -ex?
http://pubs.opengroup.org/onlinepubs/9699919799/utilities/sh.html

**Comment by Habbie** [comment]
I don't like `sh -ex` - you lose the flags when somebody does `sh ./incfiles`

**Comment by rubenk** [comment]
Excellent point! ;)

**Comment by Habbie** [comment]
Travis failed

**Pull request merged**